project.go

```diff
@@ -39,12 +39,23 @@ func (m projectModule) Register(g *gin.RouterGroup) {
 	{
 		project.GET("", m.getProjectListHandler)
 		project.POST("", m.createProjectHandler)
-		project.DELETE("/:projectId", m.deleteProjectHandler)
 		project.PUT("/:projectId", m.updateProjectHandler)
 		project.POST("/:projectId/:stage/run", m.startProjectBuildHandler)
```

Comment:
These should be removed I think, seem to be duplicate of those in the `:projectId` group below.
```suggestion
```

Reply:
Correct. Fixed